ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,101 @@
+---
+eip: unsigned
```

Comment:
```suggestion
eip: 7580
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
nice~ got it